### ISSUE TITLE
Fix global formatting for command responses

### ISSUE THREAD

**Issue by almanacks-zz** [NovelMC/NovelEngine#4]
Everything is [gray](https://github.com/NovelMC/NovelEngine/commit/9e702e03dec6566389e38d85197dd810b36abb6b). Some things are [red](https://github.com/NovelMC/NovelEngine/blob/master/src/main/java/net/novelmc/novelengine/listener/PlayerListener.java)`. 

Everything should follow proper SERVER, INFO and STAFF formats.

**Comment by falceso** [comment]
I marked the `/event` command as INFO as it seems fit, but it's also a staff only command so yeah...

~~Anyways this is a side-issue. I've reviewed `Command_permban` & `Command_unpermban` and was wondering if it was purposely coded to display only to the staff member who did it.~~ Now sending to all online staff.

Also are kick/ban messages included under INFO?

Finally, is the message always going to be grey(`&7`) or can we change it out as suited.
 
Remaining:
* [x] PlayerListener - Mixed, mostly STAFF
* [x] Staff & Architect Command
*  [x] Unpermban & PermBan - STAFF ONLY?
* [x] Event Mode? (Awaiting Reply)

**Comment by ncurran02** [comment]
@falceso the permban messages were intentional.

**Comment by falceso** [comment]
To my knowledge, command outputs are now up to format.

For future reference:
STAFF: `&8<-> &4&lSTAFF`
INFO: `&8<-> &3&lINFO`
SERVER: `&8<-> &9&lSERVER`

~~**_This issue is now safe to close._**~~

**Comment by falceso** [comment]
**New Question:** We should probably have more output into Console. I suspect a quick change of the `NUtil.globalMessage` would do, haven't tested it yet.

**Comment by ajaxvp** [comment]
I'm starting to replace old ways of getting messages out with `NUtil.globalMessage()` instead.

Also, we should be replacing `ChatColor` with `NUtil.colorize()` instead, to follow code format.

**Comment by falceso** [comment]
Yeah, good idea. Going to attach the output to console on the globalMessage system.

**Comment by almanacks-zz** [comment]
Remaining messages:
* [ ] Blocked commands
* [ ] User doesn't have permissions to use a command
* [ ] /rank
* [ ] /staff info
* [ ] /mode
* [ ] Exiting staff mode
